source/expressions.tex

```diff
@@ -433,7 +433,7 @@
 expression is called a \defn{discarded-value expression}.
 The array-to-pointer\iref{conv.array}
 and function-to-pointer\iref{conv.func} standard conversions are not
-applied. The lvalue-to-rvalue conversion\iref{conv.lval} is applied
+applied. The lvalue-to-rvalue conversion\iref{conv.lval} is performed
```

Comment:
I thought I had understand "applied", but why is this "performed", not "applied"? We do convert the lvalue to an rvalue and fail if that's not possible.

Reply:
No, we don't, per discussion on the core reflector; clarifying this was the original motivation of the change.

```
void f() {
  volatile int arr[3];
  arr;
}
```

is valid and does not apply an lvalue-to-rvalue conversion to `arr`.

Reply:
Ok.